### PR TITLE
Feat/python 3.13 the things

### PR DESCRIPTION
**Pull request by smoser**

### CODE REVIEW

File: py3-certifi.yaml [hunk 1]
```diff
@@ -24,16 +24,17 @@ vars:
...
+      3.10: '310'
+      3.11: '311'
+      3.12: '312'
+      # certifi uses a special versioning scheme where the version is a date like "2023.07.22". During the
+      # build process, the version is "normalized" (and importantly, changed) to a form like "2023.7.22",
+      # which creates problems with vulnerability scanners, since vulnerability data for certifi is filed
+      # using its actual version numbers (the "2023.07.22" form). This transform creates the "normalized"
+      # version, so that we can use it when needed to update the version string back to its original form in
+      # the Python package's metadata.
+      3.13: '313'
```
Comment (justinvreeland) [inline py3-certifi.yaml:36]: Nit: probably want to put this comment outside the list

File: py3-ply.yaml [hunk 2]
```diff
@@ -14,15 +14,16 @@ vars:
 var-transforms:
   - from: ${{package.version}}
     match: _git(.*)
-    replace: ""
+    replace: ''
```
Comment (justinvreeland) [inline py3-ply.yaml:17]: Just remove this? I guess the script caught it?

**Review by justinvreeland — COMMENTED** [review]
Looks good two nits and it'd be nice to add the test while we're here. Annoying that the automation seems to not pick the ordering up correctly.

File: py3-ply.yaml (hunk 2 above)
Comment (smoser) [reply py3-ply.yaml:17]: I dont' know why py-ruamel did that.

File: py3-certifi.yaml (hunk 1 above)
Comment (smoser) [reply py3-certifi.yaml:36]: ruamel di that too.

File: py3-ply.yaml (hunk 2 above)
Comment (justinvreeland) [reply py3-ply.yaml:17]: it removed all the quoted string so I assume it just has a rule to unquote strings and then quote this as a char? make sense to me if a bit weird.

**Comment by smoser** [comment]
I'm going to close this mega-change and do it in smaller hunks. 
c-i runs 'make package/name ' then 'make test/name'
and that is causing unrecoverable loops (py3-pip gets put to build before setuptools but its test ends up requiring it, so it fails with 'can't find 3.13').

**Comment by smoser** [comment]
closing this. almost all are in now.

**Pull request closed without merging**